truss/templates/triton/model/1/model_wrapper.py

```diff
@@ -0,0 +1,70 @@
+import importlib
+import inspect
+import sys
+from pathlib import Path
+from typing import Dict
+
+from shared.secrets_resolver import SecretsResolver
+from triton_utils import (
+    _inspect_pydantic_model,
```

Comment:
nit: prefix `_` for function names means that they're only gonna be used in the file they are defined. If you're importing them outside, please rename or refactor so a class or module can be imported instead.